pandas/core/arrays/datetimelike.py

```diff
@@ -1381,6 +1382,71 @@ def _ensure_localized(self, arg, ambiguous='raise', nonexistent='raise',
                 )
         return arg
 
+    # --------------------------------------------------------------
+    # Reductions
+
+    def _reduce(self, name, axis=0, skipna=True, **kwargs):
+        op = getattr(self, name, None)
+        if op:
+            return op(axis=axis, skipna=skipna, **kwargs)
+        else:
+            raise TypeError("cannot perform {name} with type {dtype}"
+                            .format(name=name, dtype=self.dtype))
+            # TODO: use super(DatetimeLikeArrayMixin, self)._reduce
+            #  after we subclass ExtensionArray
+
+    def min(self, axis=None, skipna=True, *args, **kwargs):
+        """
+        Return the minimum value of the Array or minimum along
```

Comment:
are these meant in inherit docstrings? these should have Parameters if not

Reply:
these are copied from the DatetimeIndexOps versions, with "Index" changes to "Array" and Index.min/Index.max added to the See Also sections.  These will be templated/shared before long hopefully

Reply:
ok can u make a follow up issue for this